src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html

```diff
@@ -247,7 +247,13 @@ <h2>
                             <fa-icon [icon]="'code-branch'" class="mr-1"></fa-icon>
                             Repository
                         </a>
-                        <button class="btn btn-primary btn-sm mr-1" jhiBuildPlanButton [projectKey]="projectKey()" [buildPlanId]="buildPlanId(value)">
+                        <button
+                            class="btn btn-primary btn-sm mr-1"
+                            [disabled]="!!buildPlanId(value)"
+                            jhiBuildPlanButton
+                            [projectKey]="projectKey()"
+                            [buildPlanId]="buildPlanId(value)!"
+                        >
```

Comment:
Something appears to be wrong with this change, since I can no longer access the button:
![grafik](https://user-images.githubusercontent.com/72132281/114908990-f153b580-9e1c-11eb-8073-806a6d099f1f.png)

Reply:
Removed behavior altering code